reactLab/src/Row.js

```diff
@@ -0,0 +1,35 @@
+import {useEffect} from "react";
+import './Row.css';
+import TextareaAutosize from 'react-textarea-autosize';
+
+function Row(props){
+    let textarea_ref = null;
```

Comment:
This doesn't look correct. See https://reactjs.org/docs/hooks-reference.html, in particular, the code for TextInputWithFocusButton.